docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,261 @@
+      * [Generate custom configuration](#generate-custom-configuration)
+      * [Build and run in the nix store](#build-and-run-in-the-nix-store)
+      * [Build and run a docker image](#build-and-run-a-docker-image)
+   * [Migrating from V0 to V1 API](#migrating-from-v0-to-v1-api)
+   * [Usage FAQs](#usage-faqs)
+      * [What are recommended hardware/software requirements for exchange wallets?](#what-are-recommended-hardwaresoftware-requirements-for-exchange-wallets)
+      * [How do I customize the wallet configuration?](#how-do-i-customize-the-wallet-configuration)
+      * [How do I export the CA certificate for the API?](#how-do-i-export-the-ca-certificate-for-the-api)
+      * [How do I know when the wallet has fetched all the blocks?](#how-do-i-know-when-the-wallet-has-fetched-all-the-blocks)
+      * [Creating a New Wallet](#creating-a-new-wallet)
+      * [Receiving Money](#receiving-money)
+      * [Sending Money](#sending-money)
+      * [Where can I find the API documentation?](#where-can-i-find-the-api-documentation)
+      * [How can I inspect runtime metrics and statistics?](#how-can-i-inspect-runtime-metrics-and-statistics)
+
+# Communication
+
+* Exchanges must provide an email address, so IOHK can broadcast issue and update announcements.
+* IOHK will create a guest Slack room to support the exchange.
+* Exchanges must provide IOHK with a status page for their service/wallets.
+
+# Requirements
+
+## Nix
+
+The wallet is built using [nix package manager](https://nixos.org/nix/). To install it on
+most Linux distros download and run the installation script.
+
+    curl https://nixos.org/nix/install > install-nix.sh
+    . install-nix.sh
+
+Follow the directions and then log out and back in.
+
+### Optional: Enable IOHK's binary cache
+
+Skip this section if you prefer to build all code from IOHK
+locally. When the binary cache is enabled build steps will tend
+go faster.
+
+    sudo mkdir -p /etc/nix
+    cat <<EOF | sudo tee /etc/nix/nix.conf
+    binary-caches            = https://cache.nixos.org https://hydra.iohk.io
+    binary-cache-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+    EOF
+
+## Miscellaneous Utilities
+
+Use `nix` to install essential utilities.
+
+    nix-env -i git
+
+# Wallet
+
+## Backup local state
+
+Skip to the next section if this is target machine doesn't yet have
+`cardano-sl` set up.
+
+To avoid catastrophic data loss, stop the wallet and backup the
+local databases, keys, certificates, and logs. By default, the
+local state will be in `./state-wallet-mainnet`, but may be
+elsewhere (see `stateDir` attribute in `./custom-wallet-config.nix`).
+
+## Fetch latest code
+
+Clone the [cardano-sl repository](https://github.com/input-output-hk/cardano-sl) or `cd` into a preexisting copy.
+
+    git clone https://github.com/input-output-hk/cardano-sl.git
+    cd cardano-sl
+
+Switch to the `master` branch and pull the latest code.
+
+    git checkout master
```

Comment:
Is it still the case with the recent changes made to the workflow? 

> rename current master branch to release/1.0.x (this is used only for exchanges)
> rename release-1.1.0 to master (latest production)

Reply:
That's a tricky one, because these docs are really going to be for master once v1 is on master, but before that, we're going to be telling people to use specific commit on develop branch.

Reply:
@disassembler @KtorZ Indeed that's tricky -- can we think about a middle ground (i.e. an interim solution) to make sure we make explicit that, up until we finally get 100% git-flow compliant, exchanges should really close the latest available `release/x.x.x` branch?